content/blog/bare-metal-provisioning-on-hpe-greenlake-using-terraform.md

```diff
@@ -0,0 +1,359 @@
+
+```hcl
+terraform {
+  required_providers {
+    hpegl = {
+      source  = "HPE/hpegl"
+      version = ">= 0.3.12"
+    }
+  }
+}
+
+provider "hpegl" {    
+  # metal block for configuring bare metal resources.
+  metal {   
+  } 
+}
+
+locals {
+  location = "USA:CO:FTC"
+}
+
+resource "hpegl_metal_ssh_key" "newssh_1" {
+  name       = "newssh_1"
+  public_key = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQCv03o//GEQ9/6eI1qZleyBbSndg0n5AkcKVnf5D4fEjwkWrtSIJEnROqJddEAn2XYALAk9x1AcB4Nue3q4tDG17VeK3ODo0+9Dx0LYqUTawnFWmo4X80QKr658Jmt7Enmnk5x2IrUDcNwAzALVellkBbwq7QbYUu1swSycNlNhSfGizqo/lQCNIHXyeRQ8oJxOuZkbiturXHZL389blIrTeUo53xmwE1TolVS8QzZRN8ve1GjFvpC5dl6orzi6LXDcrDcbZaxlrW+YQqyaipFRAw1DyTalrfpqxtq/Y9+Elz5xgCnUaepHN6ha/k81wtI2rySHga6pMOcJKlxaRS5OfzdrWh7oi2tEAaiq2y3pTr9hROQ2OGcMNU5gxbVU2ymeXdHVsAHMCmyKvQe0g0/fJzmNA/excogFCWDN7Spy9s2V39IbEKttyXjD/dpave7re9eFzYHA1CBEnNjMuvJj0H4tnpAETdQ6UbnjbE4JYn5eKGvnJ2w1JTfSdMK8nMcxqo4HfHWuLFuntCV9GAlWIVIvJn1pYisY8kEOtN5w6QrLTfsei96/TfssAsfhrDrVtgcgNU3EvZlC6Uaaly7D0ISFeufsxkPswu+jGNUJvGEqDiqvt05lSEZWS5viR/TOROTlicaGN9dhez/fqHcj5cnuoK1pmibK5GT7/Yf1Gw== user1@quattronetworks.com"
+}
+
+resource "hpegl_metal_network" "newpnet_1" {
+  name        = "newpnet_1"
+  description = "New private network 1 description"
+  location    = local.location
+  ip_pool {
+    name          = "npool"
+    description   = "New IP pool description"
+    ip_ver        = "IPv4"
+    base_ip       = "10.0.0.0"
+    netmask       = "/24"
+    default_route = "10.0.0.1"
+    sources {
+      base_ip = "10.0.0.3"
+      count   = 10
+    }
+    dns      = ["10.0.0.50"]
+    proxy    = "https://10.0.0.60"
+    no_proxy = "10.0.0.5"
+    ntp      = ["10.0.0.80"]
+  }
+}
+
+
+resource "hpegl_metal_host" "demo_advance" {
+  count = 0
+  name             = "demo-advance-1"
+  image            = "ubuntu@20.04-20210713"
+  machine_size     = "G2i"
+  ssh              = [hpegl_metal_ssh_key.newssh_1.id]
+  networks         = ["Public", hpegl_metal_network.newpnet_1.name]
+  network_route    = "Public"
+  network_untagged = hpegl_metal_network.newpnet_1.name
+  location         = local.location
+  description      = "Hello from Terraform"
+  # Attaching tags 
+  labels           = { "purpose" = "devops" }
+}
+```
+
+## Cleaning up resources   ﻿
+
+When you no longer need the resources created via Terraform, destroy the resources using the **terraform destroy** command.  This will automatically use the HPE GreenLake provider to clean the infrastructure in HPE GreenLake.
+
+![terraform destroy](/img/bmaas_terraform_cleanup.png "terraform destroy")
+
+# Summary
+
+In this blog, I covered how to provision a compute instance with Terraform provider for HPE GreenLake using bare metal resources. I also showed you advanced usage of hpegl resource statements to deploy a compute instance with dynamic resources. \
```

Comment:
I'm pretty sure you don't want the slash in here:

![image](https://user-images.githubusercontent.com/77496261/222976351-252a82fc-d322-4358-94b3-888e3a900c87.png)